corpus/phpunit.txt

```diff
@@ -2,14 +2,199 @@
+ * @codeCoverageIgnoreEnd description
+ * @codeCoverageIgnoreStart description
+ * @covers description
+ * @coversDefaultClass description
+ * @coversDefaultClass to shorten annotations description
+ * @coversNothing description
+ * @dataProvider description
+ * @depends description
+ * @depends annotation to express dependencies description
+ * @doesNotPerformAssertions description
+ * @group description
+ * @large description
+ * @medium description
+ * @preserveGlobalState description
+ * @requires description
+ * @requires usages description
+ * @runInSeparateProcess description
+ * @runTestsInSeparateProcesses description
+ * @small description
+ * @test description
+ * @testWith description
+ * @testdox description
+ * @ticket description
  */
 ---
 (document
   (tag (tag_name))
   (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag (tag_name))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
+  (tag
+    (tag_name)
+    (description (text)))
```

Comment:
I'm not opposed to testing all of these tags, but I feel like this test will be a nightmare to debug. Like, if a single tag fails to parse, won't we have no choice but to heavily edit this test and expected output just to pinpoint the failure? What about breaking these up into groups of ... maybe 5? (Or maybe there some test flags that I'm familiar w/ that highlight what parts of the input didn't parse to match the output?)

Reply:
Agreed. Change pushed.